website/static/js/contribAdder.js

```diff
@@ -373,17 +422,44 @@ var AddContributorViewModel = oop.extend(Paginator, {
             self.onInviteError.bind(self)
         );
     },
-    onInviteSuccess: function(result) {
+    onInviteSuccess: function (result) {
         var self = this;
         self.query('');
         self.results([]);
         self.page('whom');
         self.add(result.contributor);
     },
-    onInviteError: function(xhr) {
+    onInviteError: function (xhr) {
         var response = JSON.parse(xhr.responseText);
         // Update error message
         this.inviteError(response.message);
+    },
+    /**
+     * get node tree for treebeard from API V1
+     */
+    fetchNodeTree: function () {
+        var self = this;
+        var treebeardUrl = window.contextVars.node.urls.api + 'tree/';
```

Comment:
Minor: we generally try to avoid using `contextVars` in re-usable modules because they make the modules more difficult to test (you'd need to set `window.contextVars.node...` before instantiating this class). This usage is a little smelly, but not a dealbreaker.